src/app/_components/Specialists/TypeFilter.js

```diff
@@ -0,0 +1,73 @@
+'use client';
+
+import { useState } from 'react';
+import { CircularProgress } from '@mui/material';
+import { CheckBox } from '@components/CheckBox';
+import { ClearFilterButton, FilterBase } from '@components/Specialists';
+import { useSetParam, useListTherapies } from '@hooks';
+import { useSearchParams } from 'next/navigation';
+import PropTypes from 'prop-types';
+
+function TypeList({ setCount, defaultValue }) {
+  const { data: therapies, isLoading } = useListTherapies();
+  const [selectedType, setSelectedType] = useState(defaultValue);
+
+  const { addParam, deleteParam } = useSetParam('type');
+
+  const onChange = type => {
+    setSelectedType(type);
+    setCount(1);
+    addParam(type);
+  };
+
+  if (!isLoading && !therapies) return null;
+
+  if (isLoading) return <CircularProgress />;
```

Comment:
🟠 you can short that to something like
if isLoading return progress
return therapies && <ul> ....

Reply:
makes sense, I'll change it